common.go

```diff
@@ -111,3 +115,14 @@ func SplitStringMap(m map[string]string) ([]string, []string) {
 
 	return keys, values
 }
+
+// ExecCmd executes command and returns error with CombinedOutput
```

Comment:
```suggestion
// ExecCmd executes the specified Bash script. If execution fails, the error contains
// the combined output from stdout and stderr of the script.
```

Rationale: 

1. "Command" implies it's a single program, so I think it's worth mentioning this can be a whole script.
2. Since we explicitly use Bash, we should probably mention that in the comment.
3. Highlight that we don't _always_ get an error value with combined output, only on failure. (This is obviously the intent, but I like to err on the side of explicitness)

It may also be worth mentioning:

```
// As a result, you should not use this for scripts that produce a large volume of output.
```

or words to that effect.

Reply:
Done